source/databases-collections.txt

```diff
@@ -0,0 +1,286 @@
+    :end-before: end-create-collection
+
+You can specify collection options, such as maximum size and document
+validation rules, by passing them as an array to the ``createCollection()`` method.
+For a full list of optional parameters, see the `API documentation
+<{+api+}/method/MongoDBDatabase-createCollection/#parameters>`__.
+
+Get a List of Collections
+-------------------------
+
+You can query for a list of collections in a database by calling the
+``MongoDB\Database::listCollections()`` method. The method returns a
+cursor containing all collections in the database and their associated metadata.
+
+The following example calls the ``listCollections()`` method and iterates over
+the returned cursor to print the collections from the :ref:`php-db-coll-access-collection`
+and :ref:`php-db-coll-create-collection` examples:
+
+.. io-code-block::
+   :copyable:
+
+   .. input:: /includes/databases-collections/databases-collections.php
+      :language: php
+      :start-after: start-find-collections
+      :end-before: end-find-collections
+      :dedent:
+
+   .. output::
+      :language: console
+      :visible: false
+
+      MongoDB\Model\CollectionInfo Object
+      (
+         [name] => example_collection
+         [type] => collection
+         ...
+      )
+      MongoDB\Model\CollectionInfo Object
+      (
+         [name] => test_collection
+         [type] => collection
+         ...
+      )
+
+For more information about iterating over a cursor, see the :ref:`php-cursors`
+guide. 
+
+Delete a Collection
+-------------------
+
+You can delete a collection from the database by using the ``MongoDB\Database::dropCollection()``
+method.
+
+The following example deletes the ``test_collection`` collection:
+
+.. literalinclude:: /includes/databases-collections/databases-collections.php
+    :language: php
+    :dedent:
+    :start-after: start-drop-collection
+    :end-before: end-drop-collection
+
+.. warning:: Dropping a Collection Deletes All Data in the Collection
+
+   Dropping a collection from your database permanently deletes all
+   documents and all indexes within that collection.
+
+   Drop a collection only if you no longer need the data in it.
+
+.. _php-config-read-write:
+
+Configure Read and Write Operations
+-----------------------------------
+
+You can control how the library routes read operations by setting a **read preference**.
+You can also control options for how the library waits for acknowledgment of
+read and write operations on a replica set by setting a **read concern** and a
+**write concern**.
+
+By default, databases inherit read and write settings from the ``MongoDB\Client``
+instance, and collections inherit them from the database. You can change
+these settings on your database by passing an options array to the
+``MongoDB\Client::selectDatabase()``. You can change these settings on your collection
+by passing an options array to the ``MongoDB\Client::selectCollection()`` method.
```

Comment:
Regarding collection inheritance, note that if users call `Client::selectCollection()` they would effectively inherit the Client's options since an interstitial Database object is never created.

`Database::selectCollection()` isn't even mentioned in this paragraph, although it was in a previous section. I'll note that the code example only demonstrates `Database::selectCollection()`, so perhaps you want to focus on that here instead of the Client method.

I'll defer to you if any of this requires some attention.